conan/tools/env/environment.py

```diff
@@ -427,6 +427,7 @@ def loads(text):
         return result
 
 
-def register_environment_script(conanfile, path):
-    if path not in conanfile.environment_scripts:
-        conanfile.environment_scripts.append(path)
+def register_env_script(conanfile, path, group):
```

Comment:
Please, add a docstring explaining a little bit the function and its parameters.